### PR TITLE
Provide line breaks #866

### PR DESCRIPTION
**Pull request by Honk2**
Provide line breaks while entering a message and display them on display
This new behaviour is provided by a new option. It defaults to the old behaviour: send on enter

### CODE REVIEW

File: src/scripts/loqui/messenger.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ var Messenger = {
     var muc = $('section#chat')[0].dataset.muc == "true";
     var account = Accounts.current;
 
-    text = text || $('section#chat div#text').text();
+    text = text || $('section#chat div#text').html().replace(/(<br>)$/gi,"").replace(/<br>/gi,"\n").replace(/&nbsp;/gi, " ").replace(/(<([^>]+)>)/gi, "");
```
Comment (TitanNano) [inline src/scripts/loqui/messenger.js:34]: whats the purpose of `.replace(/(<br>)$/gi,"")`? Otherwise your patch looks good :+1:

**Comment by gargsms** [comment]
@TitanNano That is to remove the trailing `<br>` or next line, as WhatsApp does already.
@Honk2 Nice patch. :+1: Quite easy and elegant solution than what I tried. :stuck_out_tongue:  

EDIT: GitHub comments parse HTML written in them. So escaping the break tag

**Comment by TitanNano** [comment]
@gargsms I see!
@Honk2 can you rebase your branch into one commit then I'll be happy to merge it.

**Comment by Honk2** [comment]
Hi all. I am glad you like it. 
I will try to rebase later, does this mean to close the request and open a new one?
This is my first work on git....

**Comment by TitanNano** [comment]
no you just do a local rebase on this branch. Rebasing is basically altering your commit history.
If you search for git rebase interactive I think you should find the right thing 😉

**Pull request closed without merging**